src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Refactoring/RenameEndpointTest.cs

```diff
@@ -337,22 +365,32 @@ public async Task Handle_Rename_MultipleFileUsages()
             Assert.Equal(2, textDocumentEdit2.Edits.Length);
         }
 
+        private async Task<DocumentContext> GetDocumentContextAsync(Uri file)
```

Comment:
nit: Should we move this helper method to the bottom of the file? Seems a bit strange to have it located in between tests